common/script/index.js

```diff
@@ -1152,6 +1152,53 @@ api.wrap = function(user, main) {
             message: "+1 Gem"
           }, _.pick(user, $w('stats balance'))) : void 0;
         }
+/* buy 5 gems */
```

Comment:
This part might have been better served with a function that took 'type=='gems' and 'gems5', but for now, it's .., 'flattened code'.